dev-libs/libmateweather/libmateweather-1.26.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+fi
+
+DESCRIPTION="MATE library to access weather information from online services"
+LICENSE="LGPL-2.1+ GPL-2+"
+SLOT="0"
+
+IUSE="debug"
+
+COMMON_DEPEND=">=dev-libs/glib-2.56:2
+	>=dev-libs/libxml2-2.6:2
+	>=net-libs/libsoup-2.54:2.4
+	>=sys-libs/timezone-data-2010k:0
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-3.22:3
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
+"
+
+DEPEND="${COMMON_DEPEND}
+	dev-util/gtk-doc
```

Comment:
All of these are BDEPEND

Reply:
Thanks for the comperhensive review. I hope I understood you properly. I made the required changes and rebased all the commits.

Reply:
COMMON_DEPEND shouldn't be in BDEPEND here, I'll fix it